Events per second limits added to ossec.conf global section (#5346)
wazuh/wazuh-documentation: source/user-manual/api/configuration.rst

+------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Sub-fields | Allowed values       | Default value | Description                                                                                                                                                                                                                                                                                                                                 |
+============+======================+===============+=============================================================================================================================================================================================================================================================================================================================================+

> | allow      | yes, true, no, false | true          | Allow uploading configurations with remote commands through the Wazuh API. When this option is ``false`` it is not possible to upload **ossec.conf** files with the <command> option inside the :ref:`localfile tag <reference_ossec_localfile>` as well as the :ref:`wodle "command" option <wodle_command>`.                              |

Reviewer: ```suggestion
| allow      | yes, true, no, false | true          | Allow uploading configurations with remote commands through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files that contain the :ref:`wodle "command" option <wodle_command>` or the ``<command>`` option inside the :ref:`localfile tag <reference_ossec_localfile>`.                                 |
```

.. versionadded:: 4.4.0

+------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Sub-fields | Allowed values       | Default value | Description                                                                                                                                                                                                                                                                                                                                 |
+============+======================+===============+=============================================================================================================================================================================================================================================================================================================================================+

> | allow      | yes, true, no, false | true          | Allow uploading configurations with EPS limits through the Wazuh API. When this option is ``false`` it is not possible to upload **ossec.conf** files with <limits><eps> inside the :ref:`global tag <reference_ossec_global>`.                                                                                                             |

Reviewer: ```suggestion
| allow      | yes, true, no, false | true          | Allow uploading configurations with EPS limits through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files with ``<limits><eps>`` inside the :ref:`global tag <reference_ossec_global>`.                                                                                                                |
```
